dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/run/RunSupport.scala

```diff
@@ -114,7 +114,7 @@ private[sbt] object RunSupport {
             .find(_.severity == xsbti.Severity.Error)
             .map(CompilationException)
             .getOrElse(UnexpectedException(Some("The compilation failed without reporting any problem!"), Some(e)))
-        case e: Exception => UnexpectedException(unexpected = Some(e))
+        case e => UnexpectedException(unexpected = Some(e))
```

Comment:
I will feel a little safer with this instead

```suggestion
        case NonFatal(e) => UnexpectedException(unexpected = Some(e))
```

See same comment in https://github.com/playframework/playframework/pull/10458/files#r519905790

Reply:
@an-tex, watch-out when applying the github suggestion above as the import for `NonFatal` will be missing.

Reply:
@octonato sounds good to me and thanks for the hint :)